_dashboards/im-dashboards/notifications.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Notification settings
+parent: Index management in Dashboards
+nav_order: 60
+---
+
+# Notification settings
+
+You can configure default notification settings for index operations globally on the **Notification settings** page.  You can also configure additional notification settings for individual index operations.
+
+## Configuring default notification settings
+
+On the **Notification settings** page, you can configure the default notification settings for the following index operations that may take longer time to complete:
+
+- Open
+- Reindex
+- Split
+- Shrink
+- Clone
+- Force merge
+
+To get started, from the top menu, select **OpenSearch Plugins** > **Index Management**. Under **Index Management**, select **Notification settings**.
+
+You can choose to be notified when the operation has completed or failed. Additionally, you can select the notification channels for this notification, as shown in the following image:
+
+![Default notification settings]({{site.url}}{{site.baseurl}}/images/admin-ui-index/notifications.png)
+
+If you don't have permissions to view notification settings, you will not be able to view the default settings. 
+{: .note}
+
+## Configuring notification settings for an individual operation
+
+You can view default notification settings when you perform an indexing operation. You can also set up additional notifications. For example, if you want to configure additional notification for a reindex operation, perform the following steps:
+
+1. From the top menu, select **OpenSearch Plugins** > **Index Management**.
+1. Under **Index Management**, select **Indices**.
+1. Select the index you want to reindex.
+1. In the **Actions** dropdown list, select **Reindex**.
+1. After selecting all reindex options, expand **Advanced settings**. Under **Notifications**, default notifications are listed. 
+    
+    If you don't have permissions to view notification settings, you will not be able to view the default settings. 
+    {: .note}
```

Comment:
Insert line space after note.

Reply:
The line space doesn't matter in this case. It's spaced correctly whether or not the line is there. But I'm inserting spaces everywhere because then the list has the same spacing :)